ICSharpCode.CodeConverter/CSharp/NodesVisitor.cs

```diff
@@ -105,7 +105,7 @@ public override CSharpSyntaxNode VisitCompilationUnit(VBSyntax.CompilationUnitSy
                 }
 
                 var attributes = SyntaxFactory.List(node.Attributes.SelectMany(a => a.AttributeLists).SelectMany(ConvertAttribute));
-                var convertedMembers = node.Members.Select(m => (MemberDeclarationSyntax)m.Accept(TriviaConvertingVisitor));
+                var convertedMembers = node.Members.Select(m => (MemberDeclarationSyntax)m.Accept(TriviaConvertingVisitor)).ToReadOnlyCollection();
```

Comment:
:thumbsup:  Yeah I knew it was wrong when I did the goofy looking Skip(1).Any() in order to avoid doing a Count() that enumerates the whole list, but for some reason I didn't fix it...

Reply:
IReadOnlyCollection is my best friend